addons/website_livechat/static/src/bugfix/public_bugfix.scss

```diff
@@ -12,3 +12,9 @@
         right: $o-we-sidebar-width !important;
     }
 }
+
+.modal-open {
+    .o_livechat_button {
+        transform: translateY(100%); // prevent the livechat button from hidden the buttons of the modal if they are at the bottom of the modal
```

Comment:
- from hidden the buttons -> hidding
- the if part of the sentence seems useless